Formula/libzim.rb

```diff
@@ -0,0 +1,41 @@
+class Libzim < Formula
+  desc "Reference implementation of the ZIM specification"
+  homepage "https://github.com/openzim/libzim"
+  url "https://download.openzim.org/release/libzim/libzim-8.1.0.tar.xz"
+  sha256 "8c9bda942772bb8de1acf4832d4bda5c913d9595506187b62a35ae15e530221f"
+  license "GPL-2.0-only"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "icu4c"
+  depends_on "xapian"
+  depends_on "xz"
+  depends_on "zstd"
+
+  uses_from_macos "python" => :build
+
+  def install
+    mkdir "build" do
+      system "meson", *std_meson_args, ".."
+      system "ninja", "-v"
+      system "ninja", "install", "-v"
+    end
```

Comment:
Why not
```suggestion
    system "meson", "setup", "build", *std_meson_args
    system "meson", "compile", "-C", "build", "--verbose"
    system "meson", "install", "-C", "build"
```

Reply:
I have no opinion - just following the homebrew meson template and the libzim documentation. This seems to work though. Is there is an advantage to using meson directly?

Reply:
updating the template, https://github.com/Homebrew/brew/pull/14275

Reply:
One less build dependency as far as I know. And we don't have to create a directory to work from. I'll update the template later.